README.md

```diff
@@ -1,28 +1,32 @@
 # audio-slot-machine
-Audio Slot Machine, made with AutoIt3, <3 and keyboard.
 
-This repository is being re-built, thanks to [@meadoge](https://github.com/meadoge)
+[About](#About) | [Demo](#Demo) | [Install](#Install) | [Contributing](#Contributing) | [Licence](#Licence)
 
-# About
+## About
 This project started in 2018, inspired by [this](https://www.reddit.com/r/ProgrammerHumor/comments/6fmq09/a_simple_graphical_volume_control/).
 
 At that time, the author wanted to build a mini-game, for people who want to change their computer's sound volume in a more cooler way.
 
-# Demo
+This repository is being re-built, thanks to @khongsomeo/DT101
```

Comment:
Please make team name __bold__, thank you.

Reply:
I'll change, thanks/